sources/packages/web/src/assets/css/vuetify.scss

```diff
@@ -117,3 +117,11 @@
 .v-tab__slider {
   bottom: 5px !important;
 }
+
+.v-expansion-panel-title {
+  text-align: left;
+}
+
```

Comment:
I think we can achieve these 2 styles with 2nd last approach, https://next.vuetifyjs.com/en/components/expansion-panels/#custom-icon
![image](https://user-images.githubusercontent.com/77353155/182935489-2f68dbe4-00d7-406a-a3e6-4a3f6c37e0d2.png)
![image](https://user-images.githubusercontent.com/77353155/182935513-204c2181-286f-44d8-8be4-ea3a78d8c581.png)
![image](https://user-images.githubusercontent.com/77353155/182935545-3d283950-7491-4e8d-97d1-4f2b44e009b0.png)

Reply:
Yes, we can, that was my first attempt but in the mentioned way we need to do it for every single `v-expansion-panel`. In a current way we have way less repetitive code and also any other `v-expansion-panel` will be having the proper alignment.